src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
+
```

Comment:
Indentations are off in the newly added method. Please remember to hit auto-format shortcut after you make a change.